configure.ac

```diff
@@ -359,13 +359,19 @@ else
    AC_CHECK_HEADERS([curses.h], [],
       [AC_CHECK_HEADERS([ncurses/curses.h], [],
          [AC_CHECK_HEADERS([ncurses/ncurses.h], [],
-            [AC_CHECK_HEADERS([ncurses.h] ,[],
+            [AC_CHECK_HEADERS([ncurses.h], [],
                [AC_MSG_ERROR([can not find required ncurses header file])])])])])
 
    # check if additional linker flags are needed for keypad(3)
    # (at this point we already link against a working ncurses library)
    AC_SEARCH_LIBS([keypad], [tinfo])
 fi
+
+AC_CHECK_HEADERS([ncursesw/term.h], [],
+   [AC_CHECK_HEADERS([ncurses/term.h], [],
+      [AC_CHECK_HEADERS([term.h], [],
+         [AC_MSG_ERROR([can not find required term header file])])])])
```

Comment:
Is there a distribution providing `ncursesw/term.h` without Unicode support? If there is not this should be moved up to the above `AC_ARG_ENABLE` check (without `ncursesw/term.h` if no Unicode support was requested.

Reply:
I couldn't find any info on `ncursesw/term.h` with no Unicode support. But, I think all ncurses-related libraries will be in the same lib folder. So if `ncursesw/ncurses` is in the `AC_ARG_ENABLE` check, then probably the check for `ncursesw/term.h` should be there too. Do let me know I am wrong.